lib/src/commands/spec.dart

```diff
@@ -0,0 +1,70 @@
+
+library den.src.commands.spec;
+
+import 'dart:async';
+import 'dart:mirrors';
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:unscripted/unscripted.dart';
+import 'package:prompt/prompt.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:yaml/yaml.dart';
+
+import '../pub.dart';
+import '../theme.dart';
+
+class SpecCommand {
+
+  Pubspec _pubspec;
+  InstanceMirror get _pubspecReflection => reflect(_pubspec);
+  YamlMap get _yamlMap => _pubspec.yamlMap;
+  final fields = ['name','version','author','homepage','description'];
```

Comment:
Just noticed dart editor has author before version, let's switch to that to match.